seqr/views/utils/airflow_utils.py

```diff
@@ -0,0 +1,118 @@
+import json
+from google.auth.transport.requests import Request
+from google.oauth2 import id_token
+import requests
+
+from seqr.utils.communication_utils import safe_post_to_slack
+from seqr.utils.logging_utils import SeqrLogger
+from settings import AIRFLOW_API_AUDIENCE, AIRFLOW_WEBSERVER_URL, AIRFLOW_DAG_VERSION, \
+    SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL
+
+logger = SeqrLogger(__name__)
+
+
```

Comment:
This is a new file, but essentially everything is copied from anvil_workspace_api